packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/index.tsx

```diff
@@ -339,11 +339,9 @@ class StepFetchDevfile extends AbstractLoaderStep<Props, State> {
         variant: AlertVariant.warning,
         children: (
           <ExpandableWarning
-            textBefore="Looking for a Devfile in the git repository failed."
+            textBefore="Could not find any devfile in the Git repository"
             errorMessage={helpers.errors.getMessage(error)}
-            textAfter="The git provider may not be supported by Eclipse Che (or the git server hasn't been configured
-            appropriately). Note that the git cloning of the repository will succeed if it's public or it's private
-            and the developer git credentials secret has been configured."
+            textAfter="The Git provider is not supported."
```

Comment:
@vinokurig and @ibuziuk , I promised to get back on this earlier today, but I didn't realize that merging this was urgent.

I wonder if the message "The Git provider is not supported." can be misunderstood by users to mean that the Git provider they are using (for example, GitHub) is not supported by Che/DS.

So here's a suggestion (for another PR or to ammend this PR):
```suggestion
            textAfter="Missing configuration for Git provider. Requires administrator-configured OAuth or user-configured PAT Secret."
```